### PR TITLE
[TRIVIAL] [PDF] Fix escaped dollar sign in code

### PR DESCRIPTION
**Pull request by andralex**

### CODE REVIEW

File: latex.ddoc [hunk 1]
```diff
@@ -396,7 +396,7 @@ DDOC=% Copyright (c) 1999-$(YEAR) by Digital Mars
 \hypersetup{colorlinks=true,linkcolor=blue,urlcolor=Blue}
 
 % D language definition for the listings package
-\lstdefinelanguage{D}[ANSI]{C++}{basicstyle={\ttfamily}, basewidth={0.5em,0.5em}, breakatwhitespace=true, columns=fullflexible, commentstyle=\color{OliveGreen}\textsl, escapeinside={/*[}{]*/}, fontadjust=true, keepspaces=true, keywordstyle=\color{NavyBlue}, linewidth=\textwidth, morecomment=[n]{/+}{+/}, morekeywords={
+\lstdefinelanguage{D}[ANSI]{C++}{basicstyle={\ttfamily}, basewidth={0.5em,0.5em}, breakatwhitespace=true, columns=fullflexible, commentstyle=\color{OliveGreen}\textsl, escapeinside={/*[}{]*/}, fontadjust=true, keepspaces=true, keywordstyle=\color{NavyBlue}, linewidth=\textwidth, literate={\\\$}{\$}1, morecomment=[n]{/+}{+/}, morekeywords={
```
Comment (wilzbach) [inline latex.ddoc]: Before:

image

After:

image

**Comment by wilzbach** [comment]
FYI we also seem to be having issues when dollars are used in inline code e.g. `$(DOLLAR)`

-> https://github.com/dlang/dlang.org/pull/1994

**Comment by wilzbach** [comment]
Seems like you already added it: https://github.com/dlang/dlang.org/commit/a880389a5e6494520200a98f472ffdda623596b8#diff-15d29bbd945b9b51e50ac898ce3e7fb9 ?

**Comment by andralex** [comment]
yah, no need for this

**Pull request closed without merging**